build.bat

```diff
@@ -53,18 +53,29 @@ rem See https://learn.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-a
 set compiler_flags= %compiler_flags% /utf-8
 set compiler_defines= -DODIN_VERSION_RAW=\"%odin_version_raw%\"
 
+rem fileversion is defined as {Major,Minor,Build,Private: u16} so a bit limited
+set rc_flags=-nologo -v ^
```

Comment:
i tried to keep the output to a minimum, not sure if the `-v` here is good here, it will output
```
[vcvarsall.bat] Environment initialized for: 'x64'
Using codepage 1252 as default
Creating misc\odin.res


misc\odin.rc.
Writing VERSION:1,	lang:0x409,	size [10](https://github.com/odin-lang/Odin/pull/2600/checks#step:3:11)16.
Writing ICON:1,	lang:0x409,	size 2216
Writing ICON:2,	lang:0x409,	size [13](https://github.com/odin-lang/Odin/pull/2600/checks#step:3:14)84
Writing GROUP_ICON:101,	lang:0x409,	size 34.
Writing ICON:3,	lang:0x409,	size 22[16](https://github.com/odin-lang/Odin/pull/2600/checks#step:3:17)
Writing ICON:4,	lang:0x409,	size 1384
Writing GROUP_ICON:102,	lang:0x409,	size 34
main.cpp
libtommath.cpp
Parsing of manifest successful.
```
see https://github.com/odin-lang/Odin/pull/2600/checks